data/Templates/Stu3ToR4/RequestGroup/_Action.liquid

```diff
@@ -0,0 +1,10 @@
+{% mergeDiff msg -%}
+{
+  "prefix" : "{{msg.label}}",
+  "documentation" : [ {{ msg.documentation | to_array | batch_render: 'DataType/RelatedArtifact', 'msg' }} ],
+  "condition" : [ {{ msg.condition | to_array | batch_render: 'RequestGroup/Condition', 'msg' }} ],
+  "type" : {% include 'DataType/CodingToCodeableConcept' msg: msg.type -%},
+  "action" : [ {{ msg.action | to_array | batch_render: 'RequestGroup/action', 'msg' }} ],
```

Comment:
Same~ `RequestGroup/action` should be `RequestGroup/Action`

Reply:
Thanks! I checked all templates again to avoid lowercase. (I find that template name are case insensitive in local file system but sensitive in ACR templates. That's the reason why it not affect the result in my local tests.)
Case sensitive is not consistent in file system and ACR is a bug needs to fixed in the future. @qiwjin